msg/tools/px_generate_uorb_topic_helper.py

```diff
@@ -370,7 +370,7 @@ def rtps_message_id(msg_id_map, message):
     """
     Get RTPS ID of uORB message
     """
-    for dict in msg_id_map["rtps"]:
-        if message in dict["msg"]:
-            return dict["id"]
+    for dict in msg_id_map[0]['rtps']:
+        if message in dict['msg']:
+            return dict['id']
     return 0
```

Comment:
Should there be a warning or even exception if not found?

Reply:
That is actually something I am considering pushing in a next PR, since we currently don't have ids for all the msgs and we should.

Reply:
I will address this in the next PR.